src/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/LanguageServerWorkspaceFactory.cs

```diff
@@ -17,8 +17,8 @@
 
 namespace Microsoft.CodeAnalysis.LanguageServer.HostWorkspace;
 
-[Export(typeof(LanguageServerWorkspaceFactory)), Shared]
-internal sealed class LanguageServerWorkspaceFactory
+[Export(typeof(LanguageServerWorkspaceFactory)), Export(typeof(ILspWorkspaceFactory)), Shared]
+internal sealed class LanguageServerWorkspaceFactory : ILspWorkspaceFactory
```

Comment:
why this change?

Reply:
An interface was added to the protocol layer in order to let the misc workspace receive the same solution-level analyzers that the host workspace was getting. This may be reverted. Whether to take any changes at all the misc workspace as part of this feature work is up in the air.